doc/colvars-refman-main.tex

```diff
@@ -219,6 +233,12 @@ \subsection*{Overview}
 Each atom group is shown in a different color.
 ``Show gradient'' is available for scalar variables only.
 It creates a graphical representation of the atomic gradients of the selected variables, visualizing how the value of the collective variable would vary in response to a change in atomic coordinates.
+Vectors representing the gradient are rescaled as indicated by the radio buttons \emph{Set max. vector norm} and \emph{Set scaling factor}.
+\emph{Set max. vector norm} rescales gradients so that the largest vector component of each colvar's gradient is represented by an arrow of the specified length, in \AA.
```

Comment:
Using Å units here: is that intentional?

Reply:
Yup, the arrow is just a graphical object in VMD, the only immediate comparison is with atomic coordinates as displayed in VMD, so Angstrom seemed the best unit. I could implement unit conversion there, but there is no issue of portability to other codes, so I didn't think it was worth it.